backend/src/apiserver/main.go

```diff
@@ -235,10 +267,42 @@ func startHttpProxy(resourceManager *resource.ResourceManager) {
 	// Register a handler for Prometheus to poll.
 	topMux.Handle("/metrics", promhttp.Handler())
 
+	go startWebhookHTTPSProxy(controllerClient)
+
 	http.ListenAndServe(*httpPortFlag, topMux)
 	glog.Info("Http Proxy started")
 }
 
+func startWebhookHTTPSProxy(controllerClient ctrlclient.Client) {
```

Comment:
It's not a proxy and it's implied that it's HTTP(S).
```suggestion
func startWebhook(controllerClient ctrlclient.Client) {
```